frontend/src/views/WorkcenterView.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import  from '';
+
+describe('', () => {
+    var wrapper;
+    beforeEach(() => {
+        wrapper = shallow();
+    });
+
+    describe('#handleSubmit()', () => {
+        it('', () => {
+        });
+    });
+});
```

Comment:
This test seems to be a cut/paste test that hasn't been implemented yet.  Consider what we'd need to test to verify functionality in this unit level test.